build/wikia/transformers/transformMod.js

```diff
@@ -17,10 +17,10 @@ module.exports = async (oldMod, imageUrls) => {
     newMod = {
       name: Name,
       url: `https://warframe.fandom.com/wiki/${encodeURIComponent(Name.replace(/\s/g, '_'))}`,
-      thumbnail: imageUrls[Image],
       transmutable: Transmutable,
       introduced: Introduced
     }
+    newMod.thumbnail = imageUrls[Image]

```

Comment:
Why not set the thumbnail inside the newMod object? imageUrls can be null or undefined so it'll throw an error? If so could you make a check instead? Same thing happens in transformWarframe and transformWeapon.

Reply:
Also if you pretend to change it anyway in a later pr you can just check this as resolved.

Reply:
I don't really like them implication of saying I'll pretend to change something. It's disingenuous.

Reply:
Sorry for my english, in portuguese pretender means to intend, so i was asking if you will change it in the a future pr like you mentioned in 314

Reply:
Ah, I get what you meant now. My apologies for misunderstanding your intent. I had no plans on altering what it does right now. Given that this is a generated field and could potentially fail unreliably, I moved it out of the literal to not block the rest of it from being added

Reply:
That could be just an if checking the value instead of letting it give an error it's better by design, at least in my opinion, if you're changing the transform in a pr later we might as well do that there, what do you think? Like a refactor of the transforms?

Reply:
i'm not planning more changes specific to the transforms. I'm planning that warframes will probably have to be updated to support the multi-source pattern that weapons needed.